src/core/battle/tests.rs

```diff
@@ -1326,6 +1332,106 @@ fn throw_bomb_push() {
                         vec![effect::Knockback {
                             from: PosHex { q: 0, r: 3 },
                             to: PosHex { q: 0, r: 4 },
+                            strength: PushStrength { 0: Weight::Normal },
+                        }
+                        .into()],
+                    ),
+                    (Id(2), vec![Effect::Vanish]),
+                ],
+                timed_effects: Vec::new(),
+                scheduled_abilities: Vec::new(),
+            },
+        ],
+    );
+}
+#[test]
+fn throw_bomb_push_heavy() {
+    let prototypes = prototypes(&[
+        (
+            "thrower",
+            vec![
+                component_agent_one_attack(),
+                component_abilities(&[ability_throw_bomb_push()]),
+            ],
+        ),
+        (
+            "heavy",
```

Comment:
hmm, it's confusing that this prototype is named "heavy" but its weight is "normal".

Reply:
Indeed, the weight was not of a correct value, I'll update it to be "heavy"